src/scripts/destinyTrackerApi/reviewSubmitter.js

```diff
@@ -0,0 +1,70 @@
+import { itemTransformer } from './itemTransformer.js';
+
+class reviewSubmitter {
+  constructor($q, $http, dimPlatformService, trackerErrorHandler, loadingTracker, scoreMaintainer) {
+    this.$q = $q;
+    this.$http = $http;
+    this._itemTransformer = new itemTransformer();
+    this._trackerErrorHandler = trackerErrorHandler;
+    this._dimPlatformService = dimPlatformService;
+    this._loadingTracker = loadingTracker;
+    this._reviewDataCache = scoreMaintainer;
+  }
+
+  getReviewer() {
+    var membershipInfo = this._dimPlatformService.getActive();
```

Comment:
Is this set up to work correctly when it's initialized before the user is logged in?